### PR TITLE
(DOCSP-10793): Updated QuickStarts for Node & RN + Added Output for Node

### PR DESCRIPTION
**Pull request by mongomoe**
Jira:
----
https://jira.mongodb.org/browse/DOCSP-10793

Stage: 
------
* Node: https://docs-mongodbcom-staging.corp.mongodb.com/realm/mohammad.hunan/DOCSP-10793-QuickStart-Output-Node-RN/node/quick-start.html
* RN: https://docs-mongodbcom-staging.corp.mongodb.com/realm/mohammad.hunan/DOCSP-10793-QuickStart-Output-Node-RN/react-native/quick-start.html

Changes:
---------
* Added output of "Complete Example" for node quick start
* Changes to existing docs to adhere to new SDK changes:

  * Updated `user.identity` to `user.id`
  * Fixed `'"myPartition"'` to `"myPartition"` 
  * Added mandatory `_id` field to realm.create objects
  * Fixed filtered task (previously, the filter was on 'complete = true' but there was no complete field of the collection)
  * Fixed `object id?` to `objectId` in TaskModel

### CODE REVIEW

File: source/node/quick-start.txt [hunk 1]
```diff
@@ -220,11 +221,12 @@ To create a new Task run the method "realm.create()" inside the
 
          realm.write(() => {
            realm.create("Task", {
-             name: "buy milk, eggs, and bread",
+             _id: new mongodb.ObjectID(),
```
Comment (cbush) [inline source/node/quick-start.txt:224]: Is ObjectId no longer included in Realm?

File: source/node/quick-start.txt [hunk 2]
```diff
@@ -431,44 +435,49 @@ Run the complete example, by replacing the appId with your {+realm+} app ID.
...
  
            // Update UI in response to modified objects
            changes.modifications.forEach((index) => {
              let modifiedTask = tasks[index];
+             console.log(`modified task: ${JSON.stringify(modifiedTask, null, 2)} \n`);
              // ...
            });
-         }
  
+           // Update UI in response to deleted objects
```
Comment (cbush) [inline source/node/quick-start.txt:461]: Blocking: Isn't the order is important? Should be deletions, insertions, then updates. Otherwise, you may be working on the wrong index.

File: source/node/quick-start.txt [hunk 3]
```diff
@@ -513,8 +534,37 @@ Run the complete example, by replacing the appId with your {+realm+} app ID.
...
+
 .. ..
 ..    .. tab::
 ..       :tabid: typescript
 
 ..       .. code-block:: typescript
+
+Output
+------
+By running the above code you should get a similar output to the following:
```
Comment (cbush) [inline source/node/quick-start.txt:546]: You could tighten up the phrasing here (note: makes "running the above code" the subject, eliminating an entire clause. Makes "output" the main verb rather than "get", which aligns better with your intention, hence more clarity):

>Running the above code should output something like the following:

File: source/node/quick-start.txt (hunk 2 above)
Comment (mongomoe) [reply source/node/quick-start.txt]: Fixed now 👍

File: source/node/quick-start.txt (hunk 3 above)
Comment (mongomoe) [reply source/node/quick-start.txt:544]: 👍

File: source/node/quick-start.txt (hunk 1 above)
Comment (mongomoe) [reply source/node/quick-start.txt:224]: Nope, it's not included in Realm anymore, you have to set it manually and it throws the error `Error: Missing value for property 'Task._id'` if it's not there

Comment (cbush) [reply source/node/quick-start.txt:224]: I meant you have to import `mongodb` now?

File: source/node/quick-start.txt [hunk 4]
```diff
@@ -478,31 +485,43 @@ Run the complete example, by replacing the appId with your {+realm+} app ID.
...
                  status: "Open",
                });
              });
  
-             tasks = realm.objects("Task");
  
-             // Observe collection notifications.
-             tasks.addListener(listener);
+             // // Observe collection notifications.
+             // tasks.addListener(listener);
```
Comment (cbush) [inline source/node/quick-start.txt:512]: Why this change?

File: source/node/quick-start.txt [hunk 5]
```diff
@@ -478,31 +485,43 @@ Run the complete example, by replacing the appId with your {+realm+} app ID.
...
+                 description: "jogging pushups",
                  status: "Open",
                });
              });
  
-             tasks = realm.objects("Task");
  
-             // Observe collection notifications.
-             tasks.addListener(listener);
+             // // Observe collection notifications.
```
Comment (mongomoe) [inline source/node/quick-start.txt:511]:
```suggestion
             // Observe collection notifications.
```

File: source/node/quick-start.txt [hunk 6]
```diff
@@ -478,31 +485,43 @@ Run the complete example, by replacing the appId with your {+realm+} app ID.
...
+                 description: "jogging pushups",
                  status: "Open",
                });
              });
  
-             tasks = realm.objects("Task");
  
              // Observe collection notifications.
-             tasks.addListener(listener);
+             // tasks.addListener(listener);
```
Comment (mongomoe) [inline source/node/quick-start.txt:512]:
```suggestion
```

File: source/node/quick-start.txt [hunk 7]
```diff
@@ -478,31 +485,42 @@ Run the complete example, by replacing the appId with your {+realm+} app ID.
              };
  
              realm = await Realm.open(config);
+  
+             tasks = realm.objects("Task");
+             tasks.addListener(listener);
```
Comment (mongomoe) [inline source/node/quick-start.txt:490]:
```suggestion
             // Observe collection notifications
             tasks.addListener(listener);
```

File: source/node/quick-start.txt (hunk 4 above)
Comment (mongomoe) [reply source/node/quick-start.txt]: Accidentally commented instead of removing. Originally, `tasks.addListener` was placed after the first `realm.write` statement so it wasn't listening for it. Now I've removed it on line 512 and placed it before that first write statement
